.github/workflows/ci.yml

```diff
@@ -23,6 +23,7 @@ jobs:
         node-version: ${{ matrix.node-version }}
     - run: npm install
     - run: npm test
+    - run: npm run build
```

Comment:
For now its not really important, but shouldn't we run tests after build ? In case we want to add other tests than just the linter

Reply:
yeah you're right, changed